src/paste-markdown-html.ts

```diff
@@ -37,7 +37,8 @@ function onPaste(event: ClipboardEvent) {
   // Generate DOM tree from HTML string
   const parser = new DOMParser()
   const doc = parser.parseFromString(textHTMLClean, 'text/html')
-  const walker = doc.createTreeWalker(doc.body, NodeFilter.SHOW_ELEMENT, node =>
+  const whatToShow = navigator.userAgent.indexOf('Firefox') > -1 ? NodeFilter.SHOW_ALL : NodeFilter.SHOW_ELEMENT
```

Comment:
We should avoid user agent sniffing as it causes problems people use custom user agents. Instead we should use feature detection to determine the value of this boolean.